benchmarks/bench_ops.cpp

```diff
@@ -0,0 +1,137 @@
+#include <complex>
+#include <functional>
+#include <type_traits>
+
+#include <sycl/sycl.hpp>
+
+#include "sycl_ext_complex.hpp"
+
+#include <oneapi/dpl/random>
```

Comment:
can we avoid depending on `oneapi`? I guess it will make hipsycl people hapier

Reply:
Ya good idea. I am using it just for random gen on device, but I should probably use test fixtures, and can generate random on host and copy them over and use them for all tests, make things go faster. Just have to figure out how to do the fixtures the way I want in google bench

Reply:
Now generate random on host, so dpl not needed.